sdk/spring/spring-cloud-azure-stream-binder-servicebus/src/main/java/com/azure/spring/cloud/stream/binder/servicebus/ServiceBusMessageChannelBinder.java

```diff
@@ -165,6 +172,15 @@ protected MessageHandler getErrorMessageHandler(ConsumerDestination destination,
         };
     }
 
+    /**
+     * Dead-letters the message in this context.
```

Comment:
Let's change it to "Moves a message to the dead-letter sub-queue with dead-letter reason"